composer.json

```diff
@@ -14,12 +14,12 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
-        "silex/silex": "~1.0",
+        "php": ">=5.5",
```

Comment:
[`>=5.5.9` is correct](https://github.com/silexphp/Silex/blob/master/composer.json#L18) I think